release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -57,6 +57,34 @@ link:https://access.redhat.com/support/policy/updates/openshift[Red Hat OpenShif
 
 This release adds improvements related to the following components and concepts.
 
+[id="ocp-4-6-installation-and-upgrade"]
+=== Installation and upgrade
+
+[id="ocp-4-6-installation-osp-bare-metal-machines"]
+==== Bare metal machines for clusters on {rh-openstack}
+
+Clusters on {rh-openstack-first} can now run compute and control plane machines on bare metal. 
+
+For more information, see the {product-title} on {rh-openstack} installation documentation.
+
+[id="ocp-4-6-installation-osp-availability-zones"]
+==== {rh-openstack} availability zones selection at installation
+
+You can now select {rh-openstack} availability zones while installing a cluster on {rh-openstack}.
```

Comment:
perhaps worth to highlight they are "nova AZs"

Reply:
Done in https://github.com/openshift/openshift-docs/pull/26195/commits/d2b0ddda854770cd799bc00079e6f80f40c9ea38